bin/wp-env-config.sh

```diff
@@ -20,4 +20,13 @@ wp plugin list
 wp theme activate storefront
 wp wc customer update 1 --user=1 --billing='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}' --shipping='{"first_name":"John","last_name":"Doe","company":"Automattic","country":"US","address_1":"addr 1","address_2":"addr 2","city":"San Francisco","state":"CA","postcode":"94107","phone":"123456789"}'
 
+
+## Prepare translation for the test suite
+wp language core install nl_NL
```

Comment:
When discussing how to best approach translations, @nielslange and I talked about doing this and decided against it. The reason being that we wanted to encapsulate the tests as much as possible. We wanted to avoid adding anything here that would only be used by 1 or 2 tests. Although thinking about it now, if you guys are going to write translation tests for all the components, and use the same language, then it definitely makes sense to put them here. If we decide to go with this approach, I'll edit the `cart-checkout-translations` to use dutch as well.

Reply:
> The reason being that we wanted to encapsulate the tests as much as possible. We wanted to avoid adding anything here that would only be used by 1 or 2 tests.

@alexflorisca I totally agree with this. But looks like we're having flaky tests by installing the language from the test (Cart translation E2E tests are failed in all of my PRs, this PR too). I think installing the language in `wp-env-config.sh` is more reliable and predictable. Even if it's not a nice solution, it works most reliably for me. Also, the translation doesn't affect other tests because they all use English.

Reply:
@dinhtungdu & @alexflorisca Another way of encapsulating the languages and only activating/deactivating for certain tests, could be to mock a plugin. When facing problems implementing https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6211, @PanosSynetos suggested using a helper plugin in pca54o-3k7-p2#comment-3880. 

Instead of installing the language in `wp-env-config.sh`, we could create a mock plugin, e.g. `/tests/mocks/woo-switch-language`, and then load this plugin in `.wp-env.json` using the following code:

```json
	"plugins": [
		"./tests/mocks/woo-switch-language",
	],
```

That language switch plugin itself would look for a URL parameter, e.g. `?switch_language=nl_NL`. Once this URL parameter gets called, we can then change the site language and fetch the translations for the plugin using WordPress core functionality and/or WP-CLI.

Not saying that this is the best approach, but I wanted to share this alternative approach with you.

Reply:
Makes sense, I think CLI is a bit cleaner if we can use that, although I know in your case @nielslange it didn't work so having a plugin is a great option after that. 

I'll also just add that the translation tests aren't failing because of flakiness, they are failing for a legitimate reason, that is being worked on in #6195.

Reply:
> I'll also just add that the translation tests aren't failing because of flakiness, they are failing for a legitimate reason, that is being worked on in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6195.

Thanks for the head up! I update my test to install the language from the test. However, I don't think we need to uninstall the language, as long as we change the language back to English, i18n tests don't affect others. If we really want to uninstall the language files, we also need to remove the core, themes, and other plugins language files as well.